src/parser/GraphPatternOperation.h

```diff
@@ -31,7 +31,7 @@ class SparqlValues {
   // The variables to which the values will be bound
   std::vector<std::string> _variables;
   // A table storing the values in their string form
-  std::vector<std::vector<std::string>> _values;
+  std::vector<std::vector<TripleComponent>> _values;
```

Comment:
While we're at it:
Can you please also refactor the member `_variables` above to be of type `vector<Variable`.

Reply:
That was a surprising amount of work because of the many tests involving values.